modules/ROOT/pages/configure-service-mesh.adoc

```diff
@@ -121,20 +125,21 @@ image::create-non-mule-api-service-mesh.png[85%, 85%]
 +
 Your API is now created.
 
+[[bind-adapter]]
 == Bind your service to an API on API Manager
 
 After you provision the adapter, you must bind your service with an existing API to the adapter that you created. The adapter uses the unique API ID of the API for this binding. The Anypoint Platform uses the API ID to identify an application and maintains its configuration. The adapter uses this API ID to fetch policies from the Anypoint Platform and applies to your services that are running in the cluster.
 
 To bind the service with an API, perform the following steps:
 
 . Obtain the API ID of the API that you want to bind with the service.
-. If your applications were already running in the namespace, restart your application to inject the sidecar in the POD:
+. If your applications were already running in the namespace, restart your application to inject the sidecar in the pod:
 +
 `$ kubectl -n <namespace> patch deploy <deployname> --type=json -p='[{"op": "replace", "path": "/spec/template/metadata/labels/service-mesh.mulesoft.com","value":"enable"}]`
 +
 In the above command, replace the namespace and the deploy name with your specific details.
 +
-. Verify that the application Kubernetes service port is named following the requirements requested by Istio: https://istio.io/docs/setup/additional-setup/requirements/ .
+. Verify that the application Kubernetes service port is named following the requirements requested by Istio: https://istio.io/docs/setup/additional-setup/requirements/.
```

Comment:
Do we need all of "the application Kubernetes service port"? How about just "service port"?
Will your audience know what Istio is?